src/dropdownToggle/dropdownToggle.js

```diff
@@ -90,6 +90,16 @@ angular.module('mm.foundation.dropdownToggle', [ 'mm.foundation.position', 'mm.f
             }
           };
           $document.on('click', closeMenu);
+
+          angular.element($window).bind('resize', function() {
```

Comment:
Any reason why you use `bind()` here instead of `on()` like the rest of the event listeners?